### PR TITLE
Inline default CSS variables

### PR DESCRIPTION
**Pull request by apledger**
Refactored these components to use default inline CSS variables:

- `Accordions`
- `Button`
- `ButtonLink`
- `Card`
- `Carousel`
- `Slideshow`

Also added `showButtons` and `showScrollbar` props to the `CardCarousel` and `ProductsCarousel` components, as well as updated the CSS of these components to let off screen carousel items "peek" into the view.

### CODE REVIEW

File: apps/web/vibes/soul/primitives/accordions/index.tsx [hunk 1]
```diff
@@ -4,6 +4,27 @@ import * as AccordionsPrimitive from '@radix-ui/react-accordion';
...
+ *   --accordion-dark-title-text: hsl(var(--contrast-200));
+ *   --accordion-dark-title-text-hover: hsl(var(--background));
+ *   --accordion-dark-title-icon: hsl(var(--contrast-200));
+ *   --accordion-dark-title-icon-hover: hsl(var(--background));
+ *   --accordion-dark-content-text: hsl(var(--background));
+ *   --accordion-title-font-family: var(--font-family-mono);
+ *   --accordion-content-font-family: var(--font-family-body);
+ * }
+ * ```
+ */
```
Comment (jorgemoya) [inline apps/web/vibes/soul/primitives/accordions/index.tsx:27]: Should we add this to the docs but not the component? Same for the rest.

Comment (apledger) [reply apps/web/vibes/soul/primitives/accordions/index.tsx:27]: This is to help with tooltips in the IDE but mainly to give @agurtovoy / @fikrikarim guidance on the CSS variables to include in the updated `Theme` component. This is how @migueloller wanted to handle it.

Comment (migueloller) [reply apps/web/vibes/soul/primitives/accordions/index.tsx:27]: @jorgemoya, FWIW, I think it should be in both and ideally we drive some of the docs content from strong JSDoc comments in our components and type definitions. I personally like to keep documentation as close to the code as possible and to generate it when possible.

File: apps/web/vibes/soul/primitives/button-link/index.tsx [hunk 2]
```diff
@@ -24,13 +51,13 @@ export function ButtonLink({
...
+            'border-[var(--button-secondary-border,hsl(var(--foreground)))] bg-[var(--button-secondary-background,hsl(var(--foreground)))] text-[var(--button-secondary-foreground,hsl(var(--background)))] after:bg-[var(--button-secondary-background-hover,hsl(var(--background)))]',
           tertiary:
-            'border-[var(--button-tertiary-border)] bg-[var(--button-tertiary-background)] text-[var(--button-tertiary-foreground)] ring-[var(--button-tertiary-focus)] after:bg-[var(--button-tertiary-background-hover)]',
+            'border-[var(--button-tertiary-border,hsl(var(--contrast-200)))] bg-[var(--button-tertiary-background,hsl(var(--background)))] text-[var(--button-tertiary-foreground,hsl(var(--foreground)))] after:bg-[var(--button-tertiary-background-hover,hsl(var(--contrast-100)))]',
           ghost:
-            'border-[var(--button-ghost-border)] bg-[var(--button-ghost-background)] text-[var(--button-ghost-foreground)] ring-[var(--button-ghost-focus)] after:bg-[var(--button-ghost-background-hover)]',
```
Comment (migueloller) [inline apps/web/vibes/soul/primitives/button-link/index.tsx:33]: Did we intend to remove the focus ring? Maybe we want to use focus-visible for those?

**Review by migueloller — APPROVED** [review]

File: apps/web/vibes/soul/primitives/button-link/index.tsx (hunk 2 above)
Comment (apledger) [reply apps/web/vibes/soul/primitives/button-link/index.tsx:33]: I just moved it up and centralized it.

**Pull request merged**